tests/main/ubuntu-core-netplan-apply/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Ensure that netplan apply works on Ubuntu Core with network-setup-control
+
+details: |
+    Netplan apply is used to apply network configuration to the system
+
+environment:
+    NETPLAN: io.netplan.Netplan
+
+# only run on ubuntu core systems
+systems:
+    - ubuntu-core-*
+
+prepare: |
+    snap install test-snapd-netplan-apply --edge
+
+execute: |
+    # shellcheck source=tests/lib/journalctl.sh
+    . "$TESTSLIB/journalctl.sh"
+
+    echo "The interface is disconnected by default"
+    snap connections test-snapd-netplan-apply | MATCH 'network-setup-control +test-snapd-netplan-apply:network-setup-control +- +-'
+
+    echo "Running netplan apply without network-setup-control fails"
+    if test-snapd-netplan-apply.netplan apply; then
+        echo "Expected access denied error for netplan apply"
+        exit 1
+    fi
+
+    echo "Count how many network service restarts happened before calling netplan apply"
+    wait_for_network_config_stopped_before="$(get_journalctl_log -u systemd-networkd | grep -c 'Stopped Network Service.' || true)"
```

Comment:
This variable is a little mouthful. Perhaps just `stopped_before` and `started_before`

Reply:
ok, renamed